scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -681,9 +699,17 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
 
     @tailrec
     def countIndentInternal(pos: Int, acc: Int = 0): (Int, Int) = {
-      if (pos < 0 || scannerTokens(pos).is[LF]) (acc, pos)
-      else if (isWhitespace(scannerTokens(pos))) countIndentInternal(pos - 1, acc + 1)
-      else (-1, -1)
+      val token = scannerTokens(pos)
+      if (pos < 0) (acc, pos)
+      else
+        token match {
+          case _: LF => (acc, pos)
+          case c: Comment if token.pos.endLine > token.pos.startLine =>
```

Comment:
I think `MultilineComment` is unused, so maybe:
```suggestion
          case MultilineComment() =>
```

Reply:
I would use it but here it's needed to upcast `Token` to `Comment` that's impossible with classifier macro.
Changed it a little - extracted this check into function